cloudplatform/connectivity-dwc/src/main/java/com/sap/cloud/sdk/cloudplatform/DwcHeaderUtils.java

```diff
@@ -85,6 +89,20 @@ public static String getDwcPrincipalIdOrThrow()
         return logonName;
     }
 
+    /**
+     * This method fetches the value of the {@link #DWC_JWT_HEADER} header or throws an
+     * {@link DwcHeaderNotFoundException} if the header was not found.
+     *
+     * @return The value of the {@link #DWC_JWT_HEADER} header.
+     * @throws DwcHeaderNotFoundException
+     *             if the header was not found.
+     */
+    @Nonnull
+    public static String getDwcTokenOrThrow()
```

Comment:
[minor]

For consistency...?

```suggestion
    public static String getDwcJwtOrThrow()
```